packages/tokens/scripts/generate-css-file.js

```diff
@@ -69,6 +69,10 @@ const tokensToCss = (variables = {}, mode = "tokens") => {
           } else {
             regularVars += `  --${TELEGRAPH_VARIABLE_PREFIX}${currentPrefix}: ${value};`;
           }
+        } else if (mode === "breakpoint") {
+          if (currentPrefix.includes("breakpoint")) {
+            regularVars += `  @custom-media --${TELEGRAPH_VARIABLE_PREFIX}${currentPrefix} (min-width: ${value});`;
+          }
```

Comment:
Creates a `breakpoint.css` file that we then use in the `postcss-config` in order to be able to utilize css variable syntax with `@custom-media`.

Reply:
Yeah I can change the css file name. The idea for the tokens themselves is that they're singular, so you can reference `breakpoint-sm` instead of `breakpoints-sm`